Source/Meadow.Units/ApparentPower.cs

```diff
@@ -0,0 +1,412 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics.Contracts;
+using System.Runtime.InteropServices;
+using Meadow.Units.Conversions;
+
+namespace Meadow.Units
+{
+    /// <summary>
+    /// Represents Apparent Power 
+    /// </summary>
+    [Serializable]
+    [ImmutableObject(true)]
+    [StructLayout(LayoutKind.Sequential)]
+    public struct ApparentPower :
+        IComparable, IFormattable, IConvertible,
+        IEquatable<double>, IComparable<double>
+    {
+        /// <summary>
+        /// Creates a new `ApparentPower` object.
+        /// </summary>
+        /// <param name="value">The ApparentPower value.</param>
+        /// <param name="type">kilometers meters per second by default.</param>
```

Comment:
not `kilometers per second` but Volt Amperes.

Reply:
ah thx! I'll check all three

Reply:
fixed